sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -47,9 +43,9 @@ internal static EventDefinition GetEventDefintionFromPayload(string payload)
 
                 throw new Exception(string.Format(CultureInfo.CurrentCulture, AuthenticationEventResource.Ex_Comparable_Not_Found, comparable));
             }
-            catch (Exception ex)
+            catch (Exception)
             {
-                throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, AuthenticationEventResource.Ex_Event_Missing, ex.Message));
+                throw new InvalidOperationException(AuthenticationEventResource.Ex_Event_Missing);
```

Comment:
Should we consider setting the inner exception here to aid in potential debugging?   The lack of specificity in the `catch (Exception)` block could lead to unexpected errors looking like something you expect.

Reply:
ha, i just commented that up there. Can do.